chrome/watchWithMeBaseTest.html

```diff
@@ -0,0 +1,80 @@
+
+<!--
+Browser Action:
+
+The limitations and benefits of the Browser Action - This page does display a UI and can run scripts.
+It can't import external libraries from different URLs, meaning you have to keep any libraries locally.
+reference here: https://developer.chrome.com/extensions/contentSecurityPolicy
+
+This page can find other pages associtated with the extension and it can use the chrome APIs. The APIs
+allow for a page to access some properties, but not the DOM. This is important! This means that we need
+to use a file in a different location to access the DOM. You will learn about the contentScript in the 
+next section.
```

Comment:
It was a joke based off of online tutorials...I removed my humor from our project...I hope your happy you nazi bastard